provisioner/template.go

```diff
@@ -733,3 +745,50 @@ func sumQuantities(quantities ...string) (string, error) {
 func awsValidID(id string) string {
 	return strings.Replace(id, ":", "__", -1)
 }
+
+// instanceTypeCPU returns the vCPUs of an instance type provided as k8sresource.Quantity represented as string
+func instanceTypeCPU(context *templateContext, instanceType string) (string, error) {
+	// get the instance type info
+	instanceTypeInfo, err := context.instanceTypes.InstanceInfo(instanceType)
+
+	if err != nil {
+		return "", err
+	}
+
+	cpu := fmt.Sprintf("%v", instanceTypeInfo.VCPU)
+
+	return cpu, nil
+}
+
+// instanceTypeMemory returns the memory of an instance type provided as k8sresource.Quantity represented as string
+func instanceTypeMemory(context *templateContext, instanceType string) (string, error) {
+	// get the instance type info
+	instanceTypeInfo, err := context.instanceTypes.InstanceInfo(instanceType)
+
+	if err != nil {
+		return "", err
+	}
+
+	memory := fmt.Sprintf("%v%s", instanceTypeInfo.Memory/giga, "Gi")
```

Comment:
I think you can use https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#NewQuantity here
```go
k8sresource.NewQuantity(instanceTypeInfo.Memory, k8sresource.BinarySI)
```

Reply:
Ah, that's much better. Thanks 🙏  Added it in the next commit.

Reply:
@AlexanderYastrebov do you know of a good way to be able to return the quantity in a "friendly" way?

So, in case of memory, say I have a case of `16Gi` and I do `scaleQuantity "16Gi" 0.1`. I expect the returned quantity to look a more friendly `1.6Gi` instead of `1717986918` bytes. I'm unable to be able to use anything in the `resource` module for this. `quantity.String()` always prints the bytes, which might look ugly in manifests, no?

Reply:
I see in our existing method like `MemoryFraction` we return memory in bytes, so maybe bytes is acceptable?
https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/master/pkg/aws/instance_info_test.go#L462

Reply:
I also see when manifests are stored in etcd in k8s, the representation of memory is in `bytes`. So, I think we can keep the bytes values for now. I'll mark this as resolved now.